more_itertools/more.py

```diff
@@ -218,8 +218,8 @@ def first(iterable, default=_marker):
         return item
     if default is _marker:
         raise ValueError(
-            'first() was called on an empty iterable, and no '
-            'default value was provided.'
+            "first() was called on an empty iterable, "
```

Comment:
Please restore these quotes to their prior state.

Reply:
Will be done.

Reply:
Single quotes are back as requested.